resources/zipcodesNearMe.js

```diff
@@ -38,7 +38,11 @@ const getZipsNearMe = async (myZip, radius) => {
       return zips;
     }
   } catch (error) {
-    throw error;
+    if (error.message === 'Request failed with status code 404') {
```

Comment:
so this will just make it so they see nothing if they have an invalid zip code?

Reply:
and if an event/group has an invalid zip code, it just won't show up in any results?